pkg/ssh/config.go

```diff
@@ -87,9 +87,9 @@ func (h *host) String() string {
 
 }
 
-func newParam(keyword string, args, comments []string) *param {
+func newParam(keyword string, args, _ []string) *param {
```

Comment:
Should we remove this param if it's always nil I don't see the point of having it